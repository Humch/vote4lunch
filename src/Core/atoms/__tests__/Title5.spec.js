/* eslint-disable no-undef */
import React from 'react';

import renderer from 'react-test-renderer';


import { Title5 } from '../..';


describe('Title5 Snapshot', () => {
  const content = 'Le superbe titre';
  const color = 'white';
  const alignment = 'center';

  it('capturing Snapshot of Title5 with content only', () => {
    const renderedValue = renderer.create(
      <Title5
        content={content}
      />
    ).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });

  it('capturing Snapshot of Title5 with content and color', () => {
    const renderedValue = renderer.create(
      <Title5
        content={content}
        color={color}
      />
    ).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });

  it('capturing Snapshot of Title5 with content and alignment', () => {
    const renderedValue = renderer.create(
      <Title5
        content={content}
        alignment={alignment}
      />
    ).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
