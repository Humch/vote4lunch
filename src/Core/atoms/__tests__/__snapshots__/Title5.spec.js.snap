// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title5 Snapshot capturing Snapshot of Title5 with content and alignment 1`] = `
<h5
  className=" text-center"
>
  Le superbe titre
</h5>
`;

exports[`Title5 Snapshot capturing Snapshot of Title5 with content and color 1`] = `
<h5
  className=" text-white"
>
  Le superbe titre
</h5>
`;

exports[`Title5 Snapshot capturing Snapshot of Title5 with content only 1`] = `
<h5
  className=""
>
  Le superbe titre
</h5>
`;
